package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>quanergy_client</name>
+  <version>4.1.2</version>
+  <description>Quanergy Sensor SDK</description>
+
+  <maintainer email="ross.taylor@quanergy.com">Ross Taylor</maintainer>
```

Comment:
I took the liberty of copying the maintainer from the quanergy_client_ros repo.

I can put myself here if desired.

Reply:
this is fine